packages/components/src/ui/tooltip/test/index.js

```diff
@@ -13,20 +13,22 @@ describe( 'props', () => {
 	const baseTooltipId = 'base-tooltip';
 	const baseTooltipTriggerContent = 'WordPress.org - Base trigger content';
 	const byId = ( id ) => ( t ) => t.id === id;
-	beforeEach( () => {
+	const renderVisibleTooltip = () => {
 		render(
 			<Tooltip baseId={ baseTooltipId } content="Code is Poetry" visible>
 				<Text>{ baseTooltipTriggerContent }</Text>
 			</Tooltip>
 		);
-	} );
+	};
 
 	test( 'should render correctly', () => {
+		renderVisibleTooltip();
```

Comment:
Nitty, but I usually prefer to have the `render` call inlined into each test for better readability (and in case we needed to grab anything from return value of the render call).

Maybe we could extract a `VisibleTooltip` React component, and then refactor each test to:

```
render( <VisibleTooltip /> );
```

@Mamaduka , is that also what you were referring to?

Reply:
Correct, and I like your suggestion of extracting this into a new component on the test module level.

Reply:
That sounds like an improvement, I'm happy to follow-up with this.

Reply:
Suggested in #45105. Thanks for the feedback, both 🙌